### ISSUE TITLE
Increase the salaries crew ask for if you have a tendency to ask them to die for you.

### ISSUE THREAD

**Issue by DingusShingleton** [endless-sky/endless-sky#1650]
I just had a couple thoughts reading #1553, and I think they really have potential help balance the plundering/capturing profession and making money in general. The first is basically that the game would track the number of crew members who have died in boarding actions and leverage it to create additional costs for capturing. The second is the concept of sharing the rewards of capturing/looting with the actual crew members who are doing it.

Think about it, if I am constantly sending dozens of crew members to their deaths and paying millions in death benefits, word would quickly get around that I'm not really the kind of captain that you want to be working for, at least not for a measly 100 credits/day.

Perhaps, just for example, as your "Crew Killed" count rises, the wages that you have to pay to attract new crew members rises along with it. That could potentially hit HARD if you allowed it to rise too much, especially once you start paying hundreds or thousands of crew members. Or, a simple one time hiring bonus that would vary based on certain factors.

On the face of it, the idea that these men and women would risk their lives to get me filthy rich, all for 100 credits/day... is a little crazy. I send my crew aboard some huge capital ship, they battle it out to the death with a bunch of desperate people who are trapped like rats, and then I walk away counting my 20 million and they get diddly squat. Maybe upon capturing a ship, all the crew members who participated in the action split some percentage of the ship's value among themselves, thus simply directly lowering the profit margin by a flat rate.

Or, or, to make it a bit spicier, they get a flat bonus _per crew member_ based on the value of the ship. IE it might be a 1,000 credits/crew if you capture an interceptor but 10,000 credits/crew if you capture a light warship (just pulled the numbers out of thin air obviously). This way, it incentivizes keeping your crew complement smaller and making them more effective with hand to hand outfits (and opens the door for making more and cooler hand2hand outfits, maybe even ones that use mass) AND helps to balance the power of e.g. the almighty Bactrian's massive crew complement. Because currently, the only thing balancing capturing is death benefits, but with a strong enough crew, sustaining 0 casualties becomes a certainty and that balance goes right out the window.

It could also be applied to looting, for each outfit that is plundered the crew get a (of course much smaller because the risk is far lower) piece of the pie. In this case, a flat percentage of the value divided among the crew makes more sense I think. Combined with the depreciation system, all of a sudden there are quite a few levers that can be pulled to find that perfect balance.

On a side note, this kind of opens up the idea of deepening the Crew system in general. I mean you are hiring nameless faceless Joes who are all equally adept at flying, manning turrets, performing boarding actions, removing complex systems from ships, cleaning the space bathrooms.....I mean obviously keeping it simple stupid is certainly far preferable to the deep space equivalent of The Sims, but I feel that adding more depth to the way you manage your crew has potential to be another fun aspect of the game in some way. I'll think on this aspect more.

**Comment by Pointedstick** [comment]
Great ideas! I love 'em.

**Comment by ldsgsm** [comment]
I really like this idea, but I feel like it would be worth keeping it simple: have the game keep track of "recent crew deaths" (the way that it does "recent profit" when considering what bank loans to offer you) and scaling crew salaries based on that.

**Comment by VonZorch** [comment]
Excellent ideas, how about both.  A flat combat bonus to crew taking part in boarding, and a percentage of the value of all ships captured when you get to the nearest shipyard or fire crew.  Prize money would apply to captured ships that you keep.

**Comment by Tadrix** [comment]
Why not just give the options of hiring marines? Probably doubling salaries for extra crew (because of dangerous job), minimally increasing combat effectiveness over minimum crew.

With "reputation system", presented higher, there's a vibe that the captain is a cheating employer, hiring people and then demanding to risk their lives while they didn't signed up for it. It would warrant a mutiny. I'd like to be able to say: "Hi, I want to hire some people to do dangerous things, and I'll pay for it extra, now, who's coming?"

**Comment by VonZorch** [comment]
Mostly agree here but with some detail differences.
Marines should get a significant advantage just for being marines, +1 to both attack and defend seem appropriate.
Pay them less regular pay say 50 Cr/day but a combat bonus of Cr500 each time they engage in a boarding action either as attack or defense.  And prize money if they actually take the ship.

Perhaps expand the crew system.
Track how long each crew member  has been with you, and add watch qualifications.
Watch stations would be bridge, engineering, gunnery and boarding party.
Base pay and combat rating would vary by longevity. i.e. two crewmen with the same quals, the one who has greater longevity get paid more and has a slightly better combat rating depending on how long.
Start a green crewman out at Cr20 a day.  And you do need green crew, they're the ones that do all the unpleasant jobs that need done but don't require much skill.
When a crewman gets his first qual pay goes to Cr50, from there each additional qual would add Cr50.

Ships required crew would need to be split into types, bridge crew would be based simply on ship size and type.  And that adds another crew position, passenger service if the ship carries passengers.
Engineering would also have a base number set by ship size and type but each outfit would also have a crew required attribute, big engines take more crew than small engines.

Gunnery, you can put any greenie on a gun and teach him to fire it in a few seconds but accuracy won't be good.  That means adding a chance to hit say 25% for a greenie, 50% for a newly qualled gunner and increasing by time qualled.

I'm seeing other changes but it already looks too complicated to be worth the effort.

**Comment by KiLEdEnNis** [comment]
something relevant to this as it's based on this; different types of crew
different aliens would have different stats in boarding, and for example 6 Quarg could maybe do what 10 humans could
so adding a "crew modifier" per government (for aliens) could be an idea for the future
and then adding the option to hire combat trained human crew instead (maybe the base crew could be the same but the extra crew could be fighting crew) paired with that?

**Comment by Pointedstick** [comment]
Love those ideas, @VonZorch!

**Comment by Nechochwen-D** [comment]
@VonZorch, I have been thinking along these same lines recently. Some of these changes run parallel to how my [crew-damage](https://github.com/Nechochwen-D/endless-sky/tree/crew-damage) branch will need to be balanced.

**Comment by DingusShingleton** [comment]
Crew experience presents an excellent lever to finally get the oft-requested turret balance into the game. Gunners have a turn speed/accuracy modifier. Bam.

**Comment by Tadrix** [comment]
@VonZorch Would the game be able to remember these crew members? Because, when you sell a ship, you lose that crew members. Would it be possible to re-hire them, or just hire equally proficient ones somewhere else (for example, game offers to sell starting with equals to what were fired last)?

Salaries could be 100% for flight crew and 50% for gunners and marines. Prize money for taking a ship could be ([boarded ship's value] × [bsmc] − 3) / [bsmc]
bsmc = boarded ship's minimum crew (because that's the minimum one needs tor the new ship)
3 = captain's share
Then death benefits should be something fixed, around max. 150 K, since prize money _and_ scalable death benefits would make this unprofitable.

I'd recommend about 50 cr. for fresh crewman.
As for pay: could progression (by levels?) be something like this:
40 — 50 — 60 — 70 — 80 — 90 — 100 — 120 — 160 — 200
Also, I propose there would be some training facilities, e.g. if player wants their crew to be better prepared, crew can be trained (for a fee) to a certain level (not the highest ones, though). Perhaps, some 40%. And, probably, presence of "mentor" (most experienced crew member in the ship) could boost development speed of rookies (up to some 25% of mentor's level?).

That being said, I doubt that counting every single marine's experience would be productive. They'd just eventually die anyway, by the Rule of Random. Perhaps this could be tweaked in a way that less experienced marine dies first (and all marines before flight crews, which they're supposed to protect)

As for probability that there could at one point be too many variables for the game to calculate, my opinion is that this system should apply to flagship's crew members and individual ships, but not to _all_ the crew in the fleet. My suggestion would be this:
Game counts flying proficiency, gun proficiency and secondary weapon proficiency for ship's captain, which is a special asset installable in a ship. The player has set of these too; flagship's captain (not the player) by default stays with the ship, should player switch flagships. While on board, both of them earn experience, but considered is whichever is highest (usually player's). Captains could be hired green or experienced. Experienced from the beginning captains at equal level get higher salaries than those players raised themselves (they negotiated their pay from the start). In addition, if game wouldn't count every single marine, player could have separate "marine commander" experience (no need for marine captain, since boarding ability stays with the player).
Turret experience is accumulated by an asset (special crew member) attached to that particular type of turret (because firing a beam weapon isn't the same as firing a blaster).

So, if we need to take into account special crew members, I'd recommend to start with player (in a way), captains and gunners.

Perhaps even this (all this crew management) is too complex. Just shared my thoughts.

**Comment by VonZorch** [comment]
@Tadrix Simplest I think is to just have options to hire green or qualified crew in the hiring menu.  I suppose that additional skill level could be added but that should be random availability, weighted by the port.  Larger ports, ports with shipyards and ports on major traffic lanes would be more likely to have very skilled crew available than someplace like Trove with no yard and out in a cul-de-sac would likely have only green crew.
Come to think about it just dismissed crew should be available for rehire, with longevity, until you leave the port.
Captains for new ships would come from the most experienced crewmen, usually the First Mate of a ship in your fleet.
Crew training could be done at outfitters for a price or simply over time aboard ship.  The US Navy uses both systems, a new crewman comes out of school having trained on basics but really doesn't know that much and needs time to learn, or for some jobs they come aboard knowing nothing and learn on the job under the supervision of a skilled petty officer.
I was thinking pay would be based on total qualifications with a multiplier starting at 100% increasing each year but with sharp increases a various points.  For instance add 1% to the multiplier each year, then at year 4, when he makes petty officer add 10% and change the yearly add to 2%, then at year 10 chief petty officer,another bump of 20%.  Longevity increased would of course be only for qualified crew, the 20 year deck spacehand gets paid the same as a noob.  Maybe I should clarify a deck space hand has no skills beyond chipping paint and pushing a mop and even then they should be supervised.

@KiLEdEnNis Worthy ideas.  I'm starting to get a crewman definition, and bear with me here I am not a C programmer.  the numbers after are for a green human
struct crew
    date Hired
    int or string  Species
    float  PayMux   1
    float  Bridge  0
    float  Engineer  0
    float  Gunner  .25
    float  CloseCombat .75
    float  Steward  0
    bool  JustFired 

CloseCombat would go to 1 when the greenie gets qualed or after one year.
JustFired would be set to true if the crewman was directly fired or was on a ship that was sold.  When crew are hired starting from the oldest JustFired if true would be set to false unless the player decided to hire green replacements.  This would mean that hiring minimum crew could no longer be automatic.  when the ship lifts any crew that has JustFired true would be purger from the roster.
Races would need to be defined in a file somewhere.

**Comment by jafdy** [comment]
If you need 10 lines per crew member then the size of the save file is going to increase massively. Even if you just do the flagship it could still add some 1000 lines.

**Comment by Tadrix** [comment]
@VonZorch If we are going for percentages for fixed power levels (starting with 100%), then I'd suggest a salary progression like this:
100% — 105% — 110% — 115% — 120% — 130% — 140% — 150% — 180% — 250%…
Now, if we were to keep this simple, stupid, then just these:
Raw recruit (80%), Apprentice (90%), Able (100%), Veteran (110%), both by ability and pay.
Also, there's less code needed, we need to specify date, experience, specialty, and if active. This should shorten the save file.
No real need for anything else.
My point: Having multiple specialties makes this too complex. Draw from different rosters. You don't want as many jacks-of-all-trades as specialists anyway. Specialties could be quite few: captain, gunners, marines. Others are just filler crew with fixed salary, disregard their experience, should make things easier. If active — assume the crew stays by default with the ship when it's parked, just not getting paid (or just a captain and gunners are being paid fraction of a salary, others are laid off). Then they are on the roaster, but inactive. Same when they're fired, just then player will have the opportunity to rehire them (perhaps with different names but same exp) at any planet with shipyard (should satisfy requirement for populous enough), like Warzone 2100 does. A prospect of accidentally losing a long-timer experienced crew member forever sounds unappealing to me, even if is more realistic.

@jafdy That's why I'm proposing to consider only the player (as captain and marine commander), captains (including that of the flagship — captains stay with the ship regardless of its status, player being like second captain and admiral) and gunners. Then these values can be attached directly to player stats, and ships and turrets could reference a person attached to them (3–4 lines each, name, type (captain or gunner), time, experience.

Only, then we should exclude erratic turret fire from understaffed ship's failures if at least one of gunner's isn't dead.

**Comment by VonZorch** [comment]
@jafdy You might note that in an earlier comment I said it was getting so complicated it probably wasn't worth doing but would expand on it if asked.  Well someone asked.

**Comment by Tadrix** [comment]
@jafdy @VonZorch Yeah, I asked. Thanks.

**Comment by tehhowch** [comment]
a data structure akin to that used for fleet/stock depreciation could be used to track the number of crew killed during boarding for the past <time period>. The per-extra-crew salary could then be adjusted by the total sum over this data structure.
E.g. crew pay is `100*sum(required_crew_for_total_fleet) + (flagship.crew - flagship.requiredcrew) * (adjusted salary)` or even `100*sum(escorts_required_crew) + flagship_total_crew * adjusted_salary`
How the salary should be adjusted from the normal 100 would need discussion and balancing.

**Comment by endless-sky** [comment]
The trouble is that "death benefits" - penalizing the player for losing crew members - made it much more lucrative to prey on weak ships that pose no threat to you, rather than risking your life trying to take down a more powerful opponent. This would do exactly the same thing. That means you're penalizing more interesting, exciting game play and encouraging boring, grindy game play.

**Comment by WildLynxDev** [comment]
I agree with topicstarter.

Trading massive amounts of clueless crew into benefits for a captain seems ... wrong.
May be "Recent Death Rate" counter should be implemented.
Or even stats for each crew member - one line for each member with many values, actually including such things as race (government) and loyalty.

Or boarding should be changed - to make each ship either not boardable due to numerous, high morale crew holed up in all difficult to reach places or easily boardable due to crew's morale breakdown due to superior firepower that surrounds the disabled ship.
I find that strange, for example: Ship is disabled, it surrounded by enemy forces keeping it under gunpoint, and crew ... puts up a fight. Why? They win and ship is blown up by escorts, so they die in explosion. They lose, and they die in firefight. Few freaks may decide to die like heroes, but all 200?

**Comment by KiLEdEnNis** [comment]
if I remember, punishing players if a lot of their crew died recently was deemed quite annoying, which lead to gameplay over realism(?) (and death penalties were removed for this)
I think there was some change that allowed modifying a crew's attack & defense stats per government, but I'm not sure about this either

having some ships be unboardable would just be very annoying for players, not to mention confusing
the crew fights back as some sort of "balance", and technically they're fighting for their lives, and the game can't really check if there's a giant fleet holding them under gunpoint
if your "under gunpoint" thing went through, one could just amass a big fleet and get big-ass warships for free

in short, most things you mention are for balance and partially gameplay, not realism
deciding when balance, gameplay and realism are more important than the other 2 depends per case, and is still a bit opinionated, but I don't think the current mechanics are going to change
